collection_requirements.rst

```diff
@@ -0,0 +1,143 @@
+* The Bullhorn, a newsletter for the Ansible developer community, `back issues and how to add content <https://github.com/ansible/community/issues/546>`_
+
+Why is this needed
+===================
+
+
+Repo structure
+===============
+
+galaxy.yml
+----------
+
+* tags MUST set
+* dependencies MUST be set to ``'>=1.0.0'`` (Might have to think about dependency order when publishing)
+
+meta/runtime.yml
+----------------
+Example: `meta/runtime.yml <https://github.com/ansible-collections/collection_template/blob/main/meta/runtime.yml>`_
+* MUST define the minimim version of Ansible which this collection works with
+  * If the Collection works with Ansible 2.9, then this should be set to `>=2.9.10`
+
+Modules & Plugins
+------------------
+
+Documentation
+~~~~~~~~~~~~~~
+All module and plugin ``DOCUMENTATION`` and ``RETURN`` MUST:
+
+* Use the FQCN for ``M(...)`` See `Linking within module documentation<https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation>`_
+* Use the FQCN for ``seealso`` FIXME Docs link
+
+All module and plugin ``EXAMPLES`` MUST:
+
+* Use FQCN for module (or plugin) name.
+* for modules (or plugins) left in ansible-base use ``ansibul.builtin.template``
+
+Other items:
+* You MUST Use the FQCN for ``extends_documentation_fragment:``, unless you are referring to doc_fragments from ansible-base
+
+
+Contributor Workflow
+====================
+
+Changelogs
+~~~~~~~~~~
+
+To give a consistent feel for changelogs across collections, and ensure for collections included in the ``ansible`` package we suggest you use `antsibull-changelog <https://github.com/ansible-community/antsibull-changelog>`_
+
+
+Preferred (in descending order):
+
+1. Use antsibull-changelog (preffered)
+2. Provide ``changelogs/changelog.yaml`` in the `correct format <https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelog.yaml-format.md>`_
+3. Provide a link to the changelog file (self-hosted) (not recommended)
+
+Versioning and deprecation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+FIXME Details here
+* Deprecation vs Collection version
+* How many Collection major versions to give
+
+Repo management
+===============
+
+Repo name
+~~~~~~~~~
+
+For collections under ansible-collections the repo SHOULD be named ``NAMESPACE.COLLECTION``
+
+Branch name and configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All new repos under `ansible-collections<https://github.com/ansible-collections>`_ MUST have `main`` as the default branch.
+
+Existing repos SHOULD be converted to use ``main``
+
+Repo Protections:
+* Allow merge commits: disallowed
+
+Branch protections MUST be enforced
+* Require linear history
+* Include administrators
+
+CI Testing
+===========
+
+At a minimim ``ansible-test sanity`` MUST be run from the `latest stable ansible-base branch <https://github.com/ansible/ansible/branches/all?query=stable->`_
```

Comment:
```suggestion
At a minimim ``ansible-test sanity`` MUST be run from the `latest stable ansible-base branch <https://github.com/ansible/ansible/branches/all?query=stable->`_. We suggest to *additionally* run ``ansible-test sanity`` from the ``devel`` branch so that you find out about new linting requirements earlier.
```